Tools/Scripts/libraries/webkitbugspy/webkitbugspy/github.py

```diff
@@ -102,13 +102,21 @@ def validater(username, access_token):
             return False
 
         hostname = self.url.split('/')[2]
+        token_url = 'https://{}/settings/tokens/new'.format(hostname)
+
+        def prompt():
+            result = "GitHub's API\nProvide {} username and access token to create and update pull requests".format(hostname)
```

Comment:
Do we want to provide an option to users to skip this, and let them know that they will need to provide it next time they run a command which needs these credentials?

Reply:
The trouble in `git-webkit setup` is that failing to provide credentials here will fail setup, because we can't create a user-owned fork.

Reply:
ok. Maybe update the message being printed above to indicate that fork is being configured as well. currently it only talks about pull requests.

Reply:
We do, just not here because the webkitbugspy code is a bit more generic:
https://github.com/WebKit/WebKit/blob/1340671e6f9581528282b67320e895ebcd9e1d8b/Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/setup.py#L59